### ISSUE TITLE
Stable release?

### ISSUE THREAD

**Issue by jannejava** [SwedbankPay/swedbank-pay-sdk-php#131]
Hello! I was wondering how long it might be before we see a stable release. Could it be weeks, months, or perhaps even years away? Is there a specific release date in mind? Thank you!

**Comment by Wlundqvist** [comment]
Hello jannejava,

We have recently published the plugin called "Swedbank Pay Payment Menu" that is up to par with our latest API version. We are currently using pilots to verify the entire flow in production. In the future, this will be the release we will be working with and the Beta tags will be removed shortly and launched with support from us.

Are you an active user of one of our current versions or just interested in the release?
If you do have an active solution with us, feel free to download the new version and try it out. If you proceed with this, please let me know so we can monitor it alongside with you.

**Comment by jannejava** [comment]
Yes, I have a installation in production on the old PayEx package for a client (checkout v2?), Looking into migrating to this package since the PayEx package seems obsolete. Alos, the client want to offer more different payment options.

https://github.com/PayEx/payex-sdk-php

**Comment by Wlundqvist** [comment]
Might have been too quick to answer. This notification was pushed through our Bot for WooCommerce. Is this the plugin the customer is using or is it the SDK that you actually have questions regarding (URL you provided)?

If it is the case of the SDK, I would need to retract and get back to you with correct information.

**Comment by jannejava** [comment]
> If it is the case of the SDK, I would need to retract and get back to you with correct information. 

Yes, it's the SDK I am interrested in. It's not a WordPress/WooCommerce site.

> The SDK still only supports the V2 version of our API's and haven't been upgraded to the V3 version that is currently available.

Ok, thanks for the information! Good to know.

Just to verify we are on the same page: 
* version 3 --> https://developer.swedbankpay.com/checkout-v3/
* version 2 --> https://developer.swedbankpay.com/old-implementations/checkout-v2/

So it seems like version 2 endpoints will not go away and continue to work?

**Comment by Wlundqvist** [comment]
V2 and V3 uses the same API base, where the differences lies in the endpoints and response schemes given when information is sent to us. V3 is a more stream-lined version and cleaner in both handling and responses.

_I am looking into upgrading to version 3 for more payment options. But if this PHP SDK is not supporting version 3, it does not seems like a good idea to use yet?_
I would say that it is highly recommendable to make the switch as soon as possible. As mentioned above regarding the API base, since it's the same, the support for it will still be there for a while but no new developments will be made or added for it.

**Comment by Wlundqvist** [comment]
I would say that the likelihood of the SDK being raised to V3 by the end of the year, isn't very likely. It is something that we are working on but currently we are focusing on raising .NET and iOS/Android to V3.

**Comment by Wlundqvist** [comment]
If you do decide to upgrade to V3 by yourself, either with some or no help of the current SDK, you are always welcome to send any API-related questions to our support staff at ecomintegration@swedbankpay.com.
